src/api/app/jobs/backend_measurements_job.rb

```diff
@@ -0,0 +1,27 @@
+class BackendMeasurementsJob < ApplicationJob
+  queue_as :quick
+
+  def perform
+    return unless CONFIG['amqp_options']
+
+    RabbitmqBus.send_to_bus('metrics', "workers #{worker_status_fields}")
+
+    worker_jobs = WorkerStatusService::JobsStatisticsFetcher.call
+    worker_jobs[:waiting].each do |wj|
+      RabbitmqBus.send_to_bus('metrics', "waiting_jobs,#{jobs_fields(wj)}")
+    end
+    worker_jobs[:blocked].each do |wj|
+      RabbitmqBus.send_to_bus('metrics', "blocked_jobs,#{jobs_fields(wj)}")
```

Comment:
I'm not sure if that should land at metrics or backend_metrics. What do you think @hennevogel ?